sparse/_common.py

```diff
@@ -33,6 +36,23 @@
 )
 
 
+@numba.njit
+def nan_check(*args):
+    for i in literal_unroll(args):
+        if np.isnan(np.min(np.asarray(i))):
+            return True
+    return False
+
+
+def check_class_nan(test):
+    if isinstance(test, SparseArray):
```

Comment:
```suggestion
    if isinstance(test, (GCXS, COO)):
```

`DOK` doesn't have a `data` attribute.

Reply:
This still needs applying.